app/views/errors/unauthorized.html.erb

```diff
@@ -18,7 +18,7 @@
       </li>
     </ul>
     <div class="govuk-body">
-      <%= link_to 'Contact the POM caseload service team', contact_us_path %> if you think another problem is causing this error or <%= link_to 'go to the POM caseload service’s dashboard', root_path %>.
+      <%= link_to 'Contact the POM caseload service team', '#' %> if you think another problem is causing this error or <%= link_to 'go to the POM caseload service’s dashboard', root_path %>.
```

Comment:
What happens when user clicks _Contact the POM caseload service team_ link? Is it controlled by JS?

Reply:
OOH nice spot, that's an error.